xla/service/gpu/loop_double_buffer_transformer_test.cc

```diff
@@ -470,6 +470,57 @@ ENTRY main {
   // associated computations.
   EXPECT_EQ(while_loops_callees.size(), 8);
 }
+
+TEST_F(GpuLoopDoubleBufferTransformerTest,
```

Comment:
I suspect that the two tests above (`NestedWhileLoopRemainsFlattened*`) will start failing now. Did you check them?

Reply:
I think they should be okay. Unrolling the nested loops would only change the trip count, the number of condition and body callsites should not change?

Reply:
Hah, you're right, nevermind. I think the name of the tests above tripped me up---maybe I should change it.